### PR TITLE
Add missing descriptions to main.fmf test cases

### PR DESCRIPTION
**Pull request by msafarik**
Add descriptions to test cases that had empty or missing description field in main.fmf.

## Summary by Sourcery

Add missing descriptive metadata to various test cases defined in main.fmf files.

Documentation:
- Add human-readable descriptions to multiple existing test cases in main.fmf to clarify their purpose.

Tests:
- Update metadata of numerous Multihost, Regression, and Sanity tests by filling in previously empty or missing description fields.

### CODE REVIEW

File: Multihost/EKU-purpose/main.fmf [hunk 1]
```diff
@@ -1,4 +1,5 @@
 summary: Test for extended key usage purpose
+description: Verify that rsyslog GnuTLS streaming correctly validates Extended Key Usage (EKU) purpose in TLS certificates, accepting connections with correct EKU or no EKU and rejecting connections with wrong EKU when CheckExtendedKeyPurpose is enabled on both server and client sides.
```
Comment (prikryla) [inline Multihost/EKU-purpose/main.fmf]: @msafarik Please, intend the sentences for better readability across all your changes.

**Review by prikryla — CHANGES_REQUESTED** [review]

**Review by prikryla — APPROVED** [review]
